Tests/test_file_tiff_metadata.py

```diff
@@ -422,7 +423,8 @@ def test_too_many_entries():
     ifd.tagtype[277] = TiffTags.SHORT
 
     # Should not raise ValueError.
-    pytest.warns(UserWarning, lambda: ifd[277])
+    with pytest.warns(UserWarning):
+        _ = ifd[277]
```

Comment:
```suggestion
        ifd[277]
```

Reply:
`idf[277]` shows a `Statement seems to have no effect` warning in PyCharm, while `_ = ifd[277]` does not.

Reply:
What @nulano said – I didn't want the statement to look like a no-op (even if it has a side effect of raising a warning) and `_` is a common placeholder name, so I figured the intent is pretty clear.

Reply:
Ok. What about asserting the value instead? https://github.com/akx/Pillow/pull/1

Reply:
@radarhere Sure, that works! Merged your PR, cheers.